OpenRA.Game/Widgets/Widget.cs

```diff
@@ -24,9 +24,9 @@ public static class Ui
 		public static int LastTickTime = Game.RunTime;
 
 		static readonly Stack<Widget> WindowList = new Stack<Widget>();
+		static readonly Stack<Widget> KeyboardFocusList = new Stack<Widget>();
```

Comment:
Shouldn't these be called WindowStack and KeyboardFocusStack? Since List is another type of collection.